.github/workflows/ci.yml

```diff
@@ -1,7 +1,15 @@
 ---
 name: ci
 
-on: [push, pull_request, merge_group]
+on:
+  - push
+  - pull_request
+  - merge_group
+
+# Only build for latest push/PR unless it's main or release/
```

Comment:
This is copied from securedrop-client. So if you push to a branch, it'll cancel any running jobs and start a new one.